src/QsCompiler/Core/StatementTransformation.fs

```diff
@@ -60,7 +60,8 @@ type StatementKindTransformation(?enable) =
         let rhs = this.ExpressionTransformation stm.Rhs
         QsValueUpdate.New (lhs, rhs) |> QsValueUpdate
 
-    member private this.onPositionedBlock (intro : TypedExpression option, block : QsPositionedBlock) = 
+    abstract member onPositionedBlock : TypedExpression option * QsPositionedBlock -> TypedExpression option * QsPositionedBlock
```

Comment:
I don't think I override this method anywhere. It seems like good practice for this method to be public and overridable, like the rest of the transformation methods, in case anyone else wants to override it in the future. It's odd to have one or two methods that are private while the rest of the methods just like them are public.

Reply:
The idea why these were private was to have the walk-order for all public methods be less specific to more specific, but looks like that is rather inconvenient to keep.